apps/web/src/studio/components/workflows/node-view/WorkflowsDetailPage.tsx

```diff
@@ -37,6 +37,7 @@ export const WorkflowsDetailPage = () => {
 
   return (
     <WorkflowsPageTemplate
+      className={css({ p: 0, paddingBlockStart: 0, overflowY: 'hidden' })}
```

Comment:
Removing the padding and scroll from the parent container when rendering the workflow canvas.